### ISSUE TITLE
[CssSelector] Access node attributes

### ISSUE THREAD

**Issue by funivan** [symfony/symfony#23942]
It would be great to access node attributes. 
For example `a @href`  
How can I translate this expression to xpath?

**Comment by stof** [comment]
Well, if you want to filter based on attribute values, CSS supports it: `[href="/hello"]` (and some variants to support other comparisons than strict equality)

If you want to read the value of the attribute, this is not a CSS selector anymore.

**Comment by funivan** [comment]
@stof thanks for the fast answer. Is there any way (extend component) to implement this?

**Comment by stof** [comment]
No, because it is not the component that you should extend first, but the specification of CSS selectors.
